requirements.txt

```diff
@@ -72,6 +72,7 @@ kafka-python==2.0.2
 opensearch-py==2.0.0
 requests_aws4auth==1.1.2
 pika==1.3.1
+pymongo==4.3.3
```

Comment:
Could you add pymongo to [this list](https://github.com/mage-ai/mage-ai/blob/master/setup.py#L91) and [this list](https://github.com/mage-ai/mage-ai/blob/master/setup.py#L97) as well?